### PR TITLE
Fixed pointlight shadowmap sampling problem

### PR DESCRIPTION
**Pull request by norda97**
Pointlight were rendering their textures flipped

### CODE REVIEW

File: LambdaEngine/Source/Rendering/RenderAPI.cpp [hunk 1]
```diff
@@ -35,7 +35,7 @@ namespace LambdaEngine
 	{
 		GraphicsDeviceDesc deviceDesc = { };
 #ifdef LAMBDA_DEVELOPMENT
-		deviceDesc.Debug = true;
+		deviceDesc.Debug = false;
```
Comment (norda97) [inline LambdaEngine/Source/Rendering/RenderAPI.cpp]:
```suggestion
		deviceDesc.Debug = true;
```

**Review by norda97 — COMMENTED** [review]
Fix this

File: Assets/Shaders/Helpers.glsl [hunk 2]
```diff
@@ -405,11 +405,14 @@ float PointShadowDepthTest(vec3 fragPos, vec3 lightPos, samplerCube shadowMap, f
...
 	float currentDepth = length(fragToLight);
 
-	float shadow	= 0.0;
-	float bias		= 0.2;
-	float offset	= 0.1;
-	int samples	= 20;
-	float diskRadius = 0.05;
+	// fragToLight.x *= -1.0;
+	// fragToLight.z *= -1.0;
+	// fragToLight.y *= -1.0;
```
Comment (Ceanze) [inline Assets/Shaders/Helpers.glsl]: Remove comments

**Review by Ceanze — APPROVED** [review]

File: Assets/Shaders/Helpers.glsl (hunk 2 above)
Comment (norda97) [inline Assets/Shaders/Helpers.glsl]:
```suggestion

```

Comment (norda97) [reply Assets/Shaders/Helpers.glsl]: Im on it

Comment (norda97) [reply Assets/Shaders/Helpers.glsl]: Fixed it ;)

**Pull request merged**